tasks/natural-language-processing/sentiment-analysis/MLproject

```diff
@@ -0,0 +1,4 @@
+name: sentiment_analysis_model_env: conda.yaml
```

Comment:
Please fix the MLproject file structure.
Please refer [MLproject tutorial](https://mlflow.org/docs/latest/tutorial.html#packaging-the-training-code)